src/style.css

```
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  font-family: "Roboto";
```

Brackets are not necessary. Add additional families.
```suggestion
  font-family: Roboto, Arial, sans-serif;
```